nightwatch.conf.js

```diff
@@ -17,41 +29,64 @@ module.exports = {
   "page_objects_path": './tests/pages',
   "custom_commands_path": './tests/custom_commands',
 
-  "selenium": { // downloaded by selenium-download module (see readme)
-      "start_process": true, // tells nightwatch to start/stop the selenium process
-      "server_path": "./node_modules/nightwatch/bin/selenium.jar",
-      "host": "127.0.0.1",
-      "port": 4444, // standard selenium port
-      "cli_args": { // chrome driver is downloaded by selenium-download (see readme)
-        "webdriver.chrome.driver": "node_modules/nightwatch/bin/chromedriver"
-      }
-  },
-
   "test_settings": {
     "default": {
+      "selenium": { // downloaded by selenium-download module (see readme)
+        "start_process": true, // tells nightwatch to start/stop the selenium process
+        "server_path": "./node_modules/nightwatch/bin/selenium.jar",
+        "host": "127.0.0.1",
+        "port": 4444, // standard selenium port
+        "cli_args": { // chromedriver is downloaded by selenium-download (see readme)
+          "webdriver.chrome.driver": "./node_modules/nightwatch/bin/chromedriver"
+        }
+      },
       "filter": "tests/\*/\*.test.js",
       "screenshots": {
         "enabled": true, // if you want to keep screenshots
-        "path": './reports/screenshots', // save screenshots here
+        "path": './screenshots', // save screenshots here
```

Comment:
@4p00rv I changed this path so that everything related to testing and reports are under ./reports folder